### PR TITLE
AJM address M and D comments

### PR DESCRIPTION
**Pull request by alimanfoo**
This PR has work to address remaining comments from M and D.

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -316,54 +318,46 @@ \subsection*{Genetic backgrounds carrying resistance alleles}
 
 %%
 %
-Although it is known that pyrethroid resistance is increasing in prevalence in malaria vector populations across Africa \cite{Ranson2016}, it has not been clear whether this is being driven by the spread of resistance alleles via gene flow, by resistance alleles emerging independently in multiple locations, or by some combination of both processes.
-%
```
Comment (alimanfoo) [inline main.tex]: Chopped this to keep things lean, and stay focused on inferring adaptive gene flow.

File: main.tex [hunk 2]
```diff
@@ -316,54 +318,46 @@ \subsection*{Genetic backgrounds carrying resistance alleles}
...
 %
-If mosquitoes from different geographical locations or species carry the same resistance allele on identical or near-identical genetic backgrounds, this implies that the allele has been spread between mosquito populations by the movement and interbreeding of mosquitoes.
+In our initial report of the Ag1000G phase 1 resource \cite{Ag1000gConsortium2017}, we used 1710 biallelic SNPs from within the 73.5 kbp \textit{Vgsc} gene (1607 intronic, 103 exonic) to compute the number of SNP differences between all pairs of 1530 haplotypes derived from 765 wild-caught mosquitoes.
+%
+We then used pairwise genetic distances to perform hierarchical clustering, and found that haplotypes carrying resistance alleles in codon 995 were grouped into 10 distinct clusters, each with near-identical haplotypes.
 %
-Conversely, if the same resistance allele is found on different genetic backgrounds in different mosquito populations, this provides evidence that the allele has emerged independently in each population, either because of multiple mutational events since the introduction of pyrethroids, or because the allele was segregating at low frequency prior to the introduction of pyrethroids.
```
Comment (alimanfoo) [inline main.tex]: Also part of edits to make the lead-in shorter and stay focused.

File: main.tex [hunk 3]
```diff
@@ -316,54 +318,46 @@ \subsection*{Genetic backgrounds carrying resistance alleles}
...
-Five of these clusters carried the \texttt{L995F} allele (labelled F1-F5), and a further five clusters carried \texttt{L995S} (labelled S1-S5).
-%
-As an alternative approach, we used the same haplotype data to construct median-joining networks (Figure \ref{fig:networks}).
-%
-The network analysis allows for the reconstruction and placement of intermediate haplotypes that may not be observed in the data.
+The network analysis improves on hierarchical clustering by allowing for the reconstruction and placement of intermediate haplotypes that may not be observed in the data.
 %
 It also allows for non-hierarchical relationships between haplotypes, which may arise if recombination events have occured between haplotypes.
 %
-Furthermore, the networks allow relationships among closely-related haplotypes to be discerned.
```
Comment (alimanfoo) [inline main.tex]: This seemed unnecessary so cut.

File: main.tex [hunk 4]
```diff
@@ -316,54 +318,46 @@ \subsection*{Genetic backgrounds carrying resistance alleles}
...
-%
-We constructed these networks up to a maximum edge distance of 2 SNP differences, to ensure that each connected component in the resulting networks captures a collection of closely-related haplotypes.
+We constructed the network up to a maximum edge distance of 2 SNP differences, to ensure that each connected component captures a group of closely-related haplotypes.
 %
-The resulting networks confirmed the presence of 5 distinct groupings of haplotypes carrying \texttt{L995F}, and a further 5 haplotype groups carrying \texttt{L995S}, in close correspondence with the previous results from hierarchical clustering (97.1\% overall concordance in assignment of haplotypes to clusters).
```
Comment (alimanfoo) [inline main.tex]: I've edited the language for consistency, using "haplotype group" throughout for consistency.

File: main.tex [hunk 5]
```diff
@@ -373,22 +367,26 @@ \subsection*{Genetic backgrounds carrying resistance alleles}
...
+%
+The F4, F5 and S2 groups each contained haplotypes from both Cameroon and Gabon.
+%
+The S3 group contained haplotypes from both Uganda and Kenya.
 %
-The F4, F5 and S2 haplotypes were each found in both Cameroon and Gabon.
+The haplotypes within each of these groups were nearly identical across the entire span of the \textit{Vgsc} gene ($\pi < 5.1 \times 10^{-5}\ bp^{-1}$).
 %
-S3 haplotypes were present in both Uganda and Kenya.
+In contrast, diversity among wild-type haplotypes was two orders of magnitude greater (Cameroon \textit{An. gambiae} $\pi = 1.4 \times 10^{-3}\ bp^{-1}$; Guinea-Bissau $\pi = 5.7 \times 10^{-3}\ bp^{-1}$).
```
Comment (alimanfoo) [inline main.tex]: Have added some numbers here to illustrate the diversity contrast between selected and non-selected. I.e., expecting to find a group of haplotypes as closely related by chance is extremely unlikely.

**Comment by alimanfoo** [comment]
@cclarkson I ended up doing a bit more editing than I originally intended, but hopefully all to the good. Very happy to discuss.

File: main.tex (hunk 3 above)
Comment (cclarkson) [reply main.tex]: The "improves on" part was requested to be removed in an earlier draft by (iirc) Martin as it suggests the previous analysis was inferior and a reviewer could ask "why did we do it then?". The "alternative approach" was what was suggested instead.

Comment (alimanfoo) [reply main.tex]: I know, but as I was reshuffling a bit, it just didn't read as well. The network analysis is better than hierarchical clustering, for the reasons we give. I don't think we need to hide that. If a reviewer wants to question why we did hierarchical clustering in a previous paper, that would be a comment on the previous paper, not this one.

File: main.tex (hunk 5 above)
Comment (cclarkson) [reply main.tex]: I like it, nice.

File: main.tex (hunk 3 above)
Comment (cclarkson) [reply main.tex]: Agreed, I'm happy with this version.

**Comment by alimanfoo** [comment]
Thanks Chris. I have one more piece of work to do, which is to make the supplementary tables with the SNP data in. I'll work on that today and send you a PR asap.

**Pull request merged**